pkg/vm/manager.go

```diff
@@ -471,8 +472,34 @@ func (m *Module) Run(vm pkg.VM) error {
 	if err := m.waitAndAdjOom(ctx, vm.Name); err != nil {
 		return m.withLogs(m.logsPath(vm.Name), err)
 	}
+	for _, ifc := range vm.Network.Ifaces {
 
-	return nil
+		if ifc.NetID == "" || len(ifc.IPs) == 0 {
+			continue
+		}
+
+		stub := stubs.NewNetworkerStub(m.client)
+		namespace := stub.Namespace(ctx, ifc.NetID)
+
+		networkAddr, err := stub.GetSubnet(ctx, ifc.NetID)
+		if err != nil {
+			return errors.Wrapf(err, "failed to get network '%s'", ifc.NetID)
+		}
+
+		client := NewClient(m.socketPath(vm.Name))
+		vmData, err := client.Inspect(ctx)
+		if err != nil {
+			return errors.Wrapf(err, "failed to Inspect vm with id '%s'", vm.Name)
+		}
+
+		err = machine.StartCloudConsole(ctx, namespace, networkAddr, ifc.IPs[0], vmData.PTYPath)
+		if err != nil {
+			return errors.Wrapf(err, "failed to start cloud-console for vm id '%s'", vm.Name)
+		}
+		return nil
```

Comment:
is this meant to do an early return from the loop as soon as you started the cloud console?